pages/supervisor/supervisor.tsx

```diff
@@ -0,0 +1,123 @@
+import { Button } from '@chakra-ui/button';
+import { BsFillHouseDoorFill } from 'react-icons/bs';
+import { BiLogOut } from 'react-icons/bi';
+import Link from 'next/link';
+import { Box, SimpleGrid } from '@chakra-ui/react';
+import { styled } from 'styled-components';
+
+const StyledDiv = styled.div`
+  display: flex;
+  align-content: center;
+  justify-content: space-around;
+`;
+const StyledButton = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+export const supervisor = () => {
+  return (
+    <>
+      <title>Supervisor</title>
+      <Box
+        w='65%'
+        h='900px'
```

Comment:
@zubairahad  Can we have this gradient box for the entire page, it looks good!

Reply:
ok we can do this